### ISSUE TITLE
[openimageio] OpenImageIO Cmake rules are broken and the package is unusable

### ISSUE THREAD

**Issue by HappySeaFox** [microsoft/vcpkg#21234]
**Describe the bug**

1. When I install `openimageio`, `vcpkg` prints 
```
    (1)
    find_package(Modules CONFIG REQUIRED)
    target_link_libraries(main PRIVATE OpenImageIO::OpenImageIO)

    (2)
    find_package(OpenImageIO CONFIG REQUIRED)
    target_link_libraries(main PRIVATE OpenImageIO::OpenImageIO OpenImageIO::OpenImageIO_Util)
```

(1) is too confusing as `Modules` package doesn't exist. If I use the second approach in a client application, I see

```
CMake Error at D:/projects/git.vcpkg/installed/x64-windows/share/ilmbase/IlmBaseConfig.cmake:11 (message):
  File or directory /include referenced by variable OpenImageIO_INCLUDE_DIR
  does not exist !
Call Stack (most recent call first):
  D:/projects/git.vcpkg/installed/x64-windows/share/OpenImageIO/OpenImageIOConfig.cmake:45 (set_and_check)
  D:/projects/git.vcpkg/installed/x64-windows/share/OpenImageIO/vcpkg-cmake-wrapper.cmake:4 (_find_package)
  D:/projects/git.vcpkg/scripts/buildsystems/vcpkg.cmake:742 (include)
  CMakeLists.txt:84 (find_package)
```

**Environment**
- OS: Windows
- Compiler: MSVC 2019 x64

**To Reproduce**

**Expected behavior**

**Failure logs** 

**Additional context**

**Comment by HappySeaFox** [comment]
Mention https://github.com/microsoft/vcpkg/issues/20784

**Comment by dg0yt** [comment]
(1) is broken autodetected usage information. Added to #20190.
(2) is a missing prefix before `/include`, in the installed cmake config. IIRC there have been similar issues with other prots recently

**Comment by autoantwort** [comment]
(2) would be fixed by #20439 I think

**Comment by HappySeaFox** [comment]
Closing in favor of https://github.com/microsoft/vcpkg/pull/20439
